std/range/package.d

```diff
@@ -5217,7 +5217,7 @@ auto sequence(alias fun, State...)(State args)
 
 // iota
 /**
-   Construct a range of values that span the given starting and stopping
+   Creates a range of values that span the given starting and stopping
```

Comment:
As I'm almost as interested in natural languages as I am in programming language, I was interested in seeing if there was any [correct answer](https://english.stackexchange.com/questions/389706/is-it-ungrammatical-to-start-a-description-of-the-functionality-of-a-mechanism-w/389708#389708). Congrats, you piqued my curiosity ;-)